integration-tests/schema.prisma

```diff
@@ -72,12 +73,13 @@ model Profile {
 
 // model that just exists for testing different schema types
 model Types {
-    id       Int      @default(0)
-    bool_    Boolean  @default(false)
-    string   String   @default("")
-    integer  Int      @default(0)
-    datetime DateTime @default(now())
-    float_   Float    @default(0)
+    id          Int                    @default(0)
+    bool_       Boolean                @default(false)
+    string      String                 @default("")
+    integer     Int                    @default(0)
+    datetime    DateTime               @default(now())
+    float_      Float                  @default(0)
+    unsupported Unsupported("BIGINT")?
```

Comment:
A better test would be to have a separate model with a required `Unsupported` field, that way we can validate that `create` and `update` aren't generated but everything else still works.